### PR TITLE
Date cell support inline cell editor

### PR DESCRIPTION
**Pull request by johnhwhite**
In order to support `stopEditingWhenGridLosesFocus`, switch from using `SkyOverlayService` to using [AG Grid Popup Cell Editing](https://www.ag-grid.com/angular-grid/cell-editing/#popup) to display the calendar.

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

File: src/app/visual/edit-stop-when-loses-focus/edit-stop-when-loses-focus.component.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+
+<sky-docs-demo-control-panel-theme
+  (themeSettingsChange)="themeSettingsChange($event)"
+>
+</sky-docs-demo-control-panel-theme>
+
+<ng-container
```
Comment (Blackbaud-TrevorBurch) [inline src/app/visual/edit-stop-when-loses-focus/edit-stop-when-loses-focus.component.html:7]: What is the purpose of this container? We are showing the grid either way so the if statement seems like a noop.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor](https://developer.blackbaud.com/skyux-ag-grid?_sp.spa=pr-date-cell-support-inline-cell-editor)

**Pull request merged**